modules/virt-checking-storage-configuration.adoc

```diff
@@ -188,19 +191,25 @@ data:
     VMI "vmi-under-test-dhkb8" hotplug volume removed
   status.result.vmLiveMigration: VMI "vmi-under-test-dhkb8" migration completed
   status.result.vmVolumeClone: 'DV cloneType: "csi-clone"'
-  status.result.vmsWithNonVirtRbdStorageClass: <vm_list> # <9>
-  status.result.vmsWithUnsetEfsStorageClass: <vm_list> # <10>
+  status.result.vmsWithNonVirtRbdStorageClass: <vm_list> # <15>
+  status.result.vmsWithUnsetEfsStorageClass: <vm_list> # <16>
 ----
 <1> Specifies if the checkup is successful (`true`) or not (`false`).
 <2> The reason for failure if the checkup fails.
 <3> The time when the checkup started, in RFC 3339 time format.
 <4> The time when the checkup has completed, in RFC 3339 time format.
-<5> Specifies if there is a default storage class.
-<6> The list of golden images whose data source is not ready.
-<7> The list of golden images whose data import cron is not up-to-date.
-<8> The list of storage profiles with unknown provisioners.
-<9> The list of virtual machines that use the Ceph RBD storage class when the virtualization storage class exists.
-<10> The list of virtual machines that use an Elastic File Store (EFS) storage class where the GID and UID are not set in the storage class.
+<5> The {VirtProductName} version.
+<6> Specifies if there is a default storage class.
+<7> The list of golden images whose data source is not ready.
```

Comment:
Its also for when the datasource has no boot source(pvc/snapshot)

Reply:
Right, but not sure it's needed to be added as it's not the common case.